packages/vitest/src/types/browser.ts

```diff
@@ -2,14 +2,23 @@ import type { Awaitable } from '@vitest/utils'
 import type { WorkspaceProject } from '../node/workspace'
 import type { ApiConfig } from './config'
 
+export interface ProviderSpecificOptions {
+  webdriverio?: WebDriver.Capabilities | WebDriver.DesiredCapabilities
+  playwright?: any
```

Comment:
Couldn't find a good way here to import playwright types only

Reply:
What would be your preferred way? Would removing added types from package.json and using `unknown` for both be acceptable?

Reply:
Should be acceptable for now